-------------------------------------------------------------------------------
Test set: dao.Sql2oCategoryDaoTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1.578 sec <<< FAILURE! - in dao.Sql2oCategoryDaoTest
dao.Sql2oCategoryDaoTest  Time elapsed: 1.577 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "105.49.136.214", user "aynkrphzycrnur", database "dbgll9r0ej4bu0", SSL off
	at dao.Sql2oCategoryDaoTest.setUp(Sql2oCategoryDaoTest.java:27)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "105.49.136.214", user "aynkrphzycrnur", database "dbgll9r0ej4bu0", SSL off
	at dao.Sql2oCategoryDaoTest.setUp(Sql2oCategoryDaoTest.java:27)

dao.Sql2oCategoryDaoTest  Time elapsed: 1.578 sec  <<< ERROR!
java.lang.NullPointerException
	at dao.Sql2oCategoryDaoTest.shutDown(Sql2oCategoryDaoTest.java:39)

